packages/asc-ui/src/components/InfoCard/InfoCardStyle.ts

```diff
@@ -0,0 +1,89 @@
+import styled, { css, Theme } from '@datapunt/asc-core'
+import { svgFill, themeColor, themeSpacing } from '../../utils'
+import Button from '../Button'
+
+export enum Levels {
+  Normal,
+  Attention,
+  Error,
+}
+
+export type Props = {
+  level?: Levels
+  compact?: boolean
+  open?: boolean
+  closable?: boolean
+}
+
+type ColorMapType = Record<
+  Levels,
+  ({ theme }: { theme: Theme.ThemeInterface }) => string
+>
+
+const colorMap: ColorMapType = {
+  [Levels.Normal]: themeColor('tint', 'level3'),
+  [Levels.Attention]: themeColor('primary'),
+  [Levels.Error]: themeColor('secondary'),
+}
+
+const PADDING_H = 6
+const PADDING_V = 5
+const COMPACT_PADDING_H = 1
+const COMPACT_PADDING_V = 2
+
+export const CloseButtonWrapper = styled.div`
+  display: flex;
+  position: absolute;
+  height: 100%;
+`
+
+export const CloseButton = styled(Button)`
+  margin: auto 0;
+  background-color: transparent;
+`
+
+export default styled.div<Props>`
```

Comment:
Although this styling accomplishes the job I am not a fan of needing to use absolute positioning and a magic numbers for the height in order to get this layout working. I've spend a little time to work out a possible alternative that uses flexbox and spacing elements, see: https://codepen.io/jonkoops/pen/poJPQRN.

@chrisvanmook Perhaps we can sit down on Monday and see if we can work this out together?

Reply:
let's do that yes